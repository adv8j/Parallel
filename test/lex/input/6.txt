@Properties{@Order{
    t1->t2, t3->t4, t6->, t9;;
    t2->t6->t8, t9,t10->all;}}
