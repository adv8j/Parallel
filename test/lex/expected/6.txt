PROPERTIES
LBRACE
ORDER
LBRACE
IDENTIFIER
ARROW
IDENTIFIER
COMMA
IDENTIFIER
ARROW
IDENTIFIER
COMMA
IDENTIFIER
ARROW
COMMA
IDENTIFIER
SEMICOLON
SEMICOLON
IDENTIFIER
ARROW
IDENTIFIER
ARROW
IDENTIFIER
COMMA
IDENTIFIER
COMMA
IDENTIFIER
ARROW
ALL
SEMICOLON
RBRACE
RBRACE
